cmd/utils.go

```diff
@@ -27,15 +30,43 @@ type progDetailsOutput struct {
 }
 
 func progInfoToOutput(p *runtime.ProgInfo) (string, error) {
+	runtimeManager, err := runtime.NewManager(nil, false)
```

Comment:
the api call and runtimeManager should be in `deta details` command not here in `utils`.